### DISCUSSION TITLE
Should .zk/notebook.db be checked into git?

### DISCUSSION THREAD

**Discussion by cdimitroulas** [zk-org/zk#263] [Help] [answered]
Hi there,

I've just started using `zk` and was wondering whether .zk/notebook.db should be checked into git? I imagine it shouldn't but thought it's worth asking anyways!

**Comment by mickael-menu** [comment, marked as answer]
No you can safely ignore `.zk/notebook.db`, as it is rebuilt (if needed) when you run a `zk` command. It's only purpose is to power the querying commands.
